action/action-simulator/src/main/java/com/powsybl/action/simulator/loadflow/CaseExporter.java

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright (c) 2017, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.action.simulator.loadflow;
+
+import com.powsybl.contingency.Contingency;
+import com.powsybl.iidm.export.Exporters;
+import com.powsybl.iidm.network.Network;
+import com.powsybl.security.LimitViolation;
+
+import java.nio.file.Path;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.List;
+import java.util.Locale;
+import java.util.Objects;
+import java.util.Properties;
+import java.util.TimeZone;
+
+import org.apache.commons.lang3.StringUtils;
+
+/**
+ * @author Teofil Calin BANC <teofil-calin.banc at rte-france.com>
+ */
```

Comment:
I would prefer to also have a description phrase (what's the role of that exporter).